MOM_input

```diff
@@ -450,13 +415,15 @@ BT_PROJECT_VELOCITY = True      !   [Boolean] default = False
                                 ! use a predictor continuity step to find the pressure field, and then to do a
                                 ! corrector continuity step using a weighted average of the old and new
                                 ! velocities, with weights of (1-BEBT) and BEBT.
+DYNAMIC_SURFACE_PRESSURE = True !   [Boolean] default = False
+                                ! If true, add a dynamic pressure due to a viscous ice shelf, for instance.
```

Comment:
Do we need this? It seems to be useful only for a dynamic ice shelf, which we don't have (yet).
I'm imagine "dynamic pressure" here is akin to that defined [in Fig 1 here](https://www.cambridge.org/core/journals/journal-of-glaciology/article/model-of-viscoelastic-iceshelf-flexure/2F701D2F33A819BFBB018AA7E421238C). I guess turning this off would also require USE_RIGID_SEA_ICE = False? See comment on lines +665 to +677 below.
```suggestion
```

Reply:
@claireyung should the ice shelf be hydrostatic, rigid, or dynamically relax from rigid to hydrostatic over some timescale?

Reply:
OK, so assuming it has no stability or performance penalties we can just leave it on even in the configs without ice shelves.